backend/src/main/java/com/springBoot/fruits_ecommerce/models/AuthenticationRequest.java

```diff
@@ -0,0 +1,22 @@
+package com.springBoot.fruits_ecommerce.models;
+import jakarta.validation.constraints.Email;
+import jakarta.validation.constraints.NotBlank;
+import jakarta.validation.constraints.NotNull;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@Builder
+@AllArgsConstructor
+@NoArgsConstructor
+public class AuthenticationRequest {
+    @NotBlank(message = "email is required")
```

Comment:
please maintain proper alignment for Annotations here, and add whitespaces (New lines) between properties